### PR TITLE
BLEN-52: Improve UsdImagingLite project. BLEN-41: Improve PreviewEngine

### PR DESCRIPTION
**Pull request by bnagirniak**
### PURPOSE
UsdImagingLite project is obsolete and doesn't correspond to current requirements for our project. It doesn't support stage changes and could cause some crashes.

### EFFECT OF CHANGE
Improved rendering process for final render. 
Improved material preview, now it is much more responsive during material changes or rendering scene changes.

### TECHNICAL STEPS
1. Rewritten UsdImagingLite project: improved/cleaned up code, fixed bugs, improved stability, added functionality.
2. Implemented static renderer usage in PreviewEngine, which is removed by lifetime.

CIS:REBUILD_DEPS

### CODE REVIEW

File: deps/UsdImagingLite/pxr/usdImaging/usdImagingLite/engine.cpp [hunk 1]
```diff
@@ -165,100 +91,80 @@ bool UsdImagingLiteEngine::SetRendererAov(TfToken const &id)
...
-    SdfPath renderBufferId = _taskDataDelegate->GetDelegateID().AppendElementString("aov_" + id.GetString());
+    SdfPath renderBufferId = _renderDataDelegate->GetDelegateID().AppendElementString("aov_" + id.GetString());
     HdRenderBuffer *rBuf = static_cast<HdRenderBuffer*>(_renderIndex->GetBprim(HdPrimTypeTokens->renderBuffer, renderBufferId));
     void *data = rBuf->Map();
     memcpy(buf, data, rBuf->GetWidth() * rBuf->GetHeight() * HdDataSizeOfFormat(rBuf->GetFormat()));
     rBuf->Unmap();
     return true;
 }
 
-UsdImagingGLRendererSettingsList UsdImagingLiteEngine::GetRendererSettingsList() const
```
Comment (DagerD) [inline deps/UsdImagingLite/pxr/usdImaging/usdImagingLite/engine.cpp:176]: Why did you completely remove this method?

Comment (bnagirniak) [reply deps/UsdImagingLite/pxr/usdImaging/usdImagingLite/engine.cpp:176]: We don't use it anywhere, also it is available in UsdImagingGLEngine.

**Review by DagerD — APPROVED** [review]
Tested. Works much better.

**Review by VascoPi — APPROVED** [review]
Works a lot faster. Looks good.

**Comment by bnagirniak** [comment]
Ready to merge.

**Pull request merged**